examples.org

```diff
@@ -292,10 +292,10 @@ Don't map a to b when there are three fingers on the trackpad.
                       :sim     250
                       ;; to_delayed_action_delay_milliseconds
                       ;; checkout karabiner's documentation
-                      ;; https://pqrs.org/osx/karabiner/json.html
+                      ;; https://karabiner-elements.pqrs.org/docs/json/complex-modifications-manipulator-definition/to-delayed-action/
                       ;; basically it means time after which the key press is count delayed
                       :delay   500
-                      ;; https://pqrs.org/osx/karabiner/json.html#to-if-alone
+                      ;; https://karabiner-elements.pqrs.org/docs/json/complex-modifications-manipulator-definition/to-if-alone/
```

Comment:
The old URL was pointing to the same JSON spec mentioned [here](https://github.com/yqrashawn/GokuRakuJoudo/pull/255/files#diff-54d121b9f8f94af2cb3fa5d2bbb327559d7145a8e5f45a98cc944cf204247cd8L22
), but I believe it was meant to point to the `to-delayed-action` doc which I've changed it to, but please let me know if I'm mistaken.